Makefile

```diff
@@ -1,4 +1,4 @@
-version="2.4.0"
+version="2.4.1"
```

Comment:
Please run `make deb-arm` on your machine and commit the changes, so we have the version updated in all the places.